src/status_im/contexts/wallet/events.cljs

```diff
@@ -32,8 +32,7 @@
 (rf/reg-event-fx :wallet/navigate-to-new-account
  (fn [{:keys [db]} [address]]
    {:db (assoc-in db [:wallet :current-viewing-account-address] address)
-    :fx [[:dispatch [:hide-bottom-sheet]]
```

Comment:
do we definitely need to move this to the other event?

Reply:
Yes, we agreed to close the bottom sheet before the navigation happens

Reply:
Right,  but the original order did that no?